docs/api/platform/Error.md

```diff
@@ -130,6 +130,37 @@ Mbed OS application and system developers may need to define error codes specifi
+
+#### Adding Crash-data-RAM region for crash reporting
+As mentioned above, the crash reporting feature requires a special memory region, called Crash-data-RAM to work. This region is 256 bytes in size and is allocated using linker scripts for the target for each toolchain. Although all platforms support crash reporting feature, not all targets are currently modified to allocate this Crash-data-RAM region.
+See `mbed_lib.json` in platform directory to see which targets currently enabled with crash reporting. In order to enable crash reporting in other targets, you must modify the linker scripts for those targets to allocate the Crash-data-RAM region. You may refer the linker scripts for one of the targets already enabled with crash reporting to understand how the Crash-data-RAM region is allocated. Below are some guidelines to make the linker script changes.
+
+* The region size should be 256 bytes and aligned at 8-byte offset.
+* If you are enabling the Crash-data-RAM for *ARM compiler*, linker script must export the following symbols:
+__Image$$RW_m_crash_data$$ZI$$Base__ - Indicates start address of Crash-data-RAM region.
+__Image$$RW_m_crash_data$$ZI$$Size__ - Indicates size of Crash-data-RAM region.
+* If you are enabling the Crash-data-RAM for *GCC ARM compiler* or *IAR Compiler*, linker script must export the following symbols:
+__\_\_CRASH_DATA_RAM_START\_\___ - Indicates start address of Crash-data-RAM region.
+__\_\_CRASH_DATA_RAM_END\_\___ - Indicates end address of Crash-data-RAM region.
+
+It's important that these regions should be marked with appropriate attributes(based on toolchain) to mark them as uninitialized region. For example, for ARM Compiler Crash-data-RAM can be marked with attribute *EMPTY*. There is no hard requirement about the placement of this region. The only requirement is that it should be placed such that no other entity is overwriting this region when rebooted or at runtime. But in order to avoid fragmentation its best placed just after the vector table region, or if there is no vector table region, it can be placed at the bottom of RAM(lowest address).
+See [memory model](memory.html) for more info on the placement of this region.
+
+#### Configuring crash reporting and auto-reboot
+MbedOS crash reporting implementation provides many options to configure the crash reporting behavior.
+Below is the list of new configuration options available to configure crash reporting functionality. These configuration options are defined in `mbed_lib.json` under platform directory.
+
+`crash-capture-enabled` - Enables crash context capture when the system enters a fatal error/crash. When this is disabled it also disables other dependent options mentioned below.
```

Comment:
Move the configuration options laid out here into a list (Github proposed changes beta feature doesn't support multiline changes yet):

"Below is the list....."
* `crash-capture-enabled` - Enables crash context capture when the system enters a fatal error/crash. When this is disabled it also disables other dependent options mentioned below.
* `fatal-error-auto-reboot-enabled` - Setting this to true enables auto-reboot on fatal errors.
* `reboot-crash-report-enabled` - Enables crash report printing over terminal when the system reboots after a fatal error. The format of this report is identical to error reporting structure mentioned in [error report](#error-reporting).
* `error-reboot-max` - Maximum number of auto-reboots permitted on fatal errors. The system will stop auto-rebooting once the maximum limit is reached. Setting this to value 0 will disable auto-reboot.